resume.html

```diff
@@ -0,0 +1,167 @@
+}
+text {
+    font-family: "Lato", sans-serif;
+}
+h1 {
+    font-family: "Lato";
+	font-size: 3.5em;
+	text-indent: 0px;
+}
+h2 {
+    font-family: "Lato";
+	font-size: 2.2em;
+    text-indent: 0px;
+}
+h3 {
+    font-family: "Lato";
+    font-size: 1.3em;
+    text-indent: 0px;
+}
+p {
+    font-family: "Lato";
+    font-size: 1em;
+}
+.panel h3 {
+  color: #395C6B;
+}
+.panel p {
+  color: #395C6B;
+}
+.accordion {
+  background-color: #E3BAC6;
+  color: #596475;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.5s;
+}
+.active, .accordion:hover {
+  background-color: #395C6B;
+  color: white;
+}
+
+.panel {
+  padding-left: 5vw;
+  padding-right: 5vw;
+  background-color: #B0D0D3;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+}
+</style>
+    </head>
+
+    <body> 
+        <div class="TopBar">
+        </div>
+        <div class="Sticky">
+          <h1> Rossy Yang </h1>
+          <h2> (631) 988-3820 </h2>
+          <h2> rossyyang@gmail.com </h2>
+        </div>
+        <div class="Main">
+         <button class="accordion"> Education </button>
+            <div class="panel">
+              <h3> MICA </h3>
+              <h3> Syosset High School </h3>
+            </div>
+          <button class="accordion"> Experience </button>
+            <div class="panel">
+              <h3> EdConnective </h3>
+              <p> Completes tasks assigned by the COO, including but not limited to scheduling calls, interviews, and meetings; writing emails and internal documents; creating and maintaining graphic assets; researching; running payroll; and office management. </p>
+              <h3> Chengshi Inc. </h3>
+              <p> Created logos and web assets for the company’s founder. Assisted in filing paperwork, record keeping, as well as client outreach via email. Most recently, planned curriculum for remote classes for young artists. </p>
+              <h3> Freelance </h3>
+              <p> Assessing aesthetic needs of various companies and individuals and working with them to create visual languages that fit their brand and company image. </p>
+            </div>
+          <button class="accordion"> Skills </button>
+            <div class="panel">
+              <h3> Multitasking 
+                <br/> Detail Oriented 
+                <br/> Resourceful
+                <br/> Efficient
+                <br/> Tech Savvy
+                <br/> Microsoft Office
+                <br/> G-Suite
+                <br/> Adobe Suite
+                <br/> Personable
+                <br/> Creative
+                <br/> Planning
+                <br/> Record-keeping
+                <br/> Project Management
+                <br/> Communication
+                <br/> Fluent in Chinese
+                <br/> Proficient in French
+                <br/> Proficient in Japanese
+                <br/> Ok at English :/ 
+              </h3>
```

Comment:
The `br` element generally should be reserved for meaningful line breaks, like in an address or poem; see [MDN's docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) for more info. This should be an unordered list (`ul`) element instead of an `h3` broken up by `br`s.